### PR TITLE
fix: Incorrect chart values for user, group and memory limit 

### LINKED ISSUE #343: Incorrect chart values for user, group and memory limit
### What happened?

I found some values that I think are incorrect and cause some problems:
- Write error because on the uid and gid incorrect
- Memory limit too big

### How can we reproduce this?

Deploy the chart without override memory limits and securityContext runAsGroup and runAsUser

### Helm Chart version

```bash
0.32.1
```

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by gianmarco-mameli**
Closes #343 

Fixed chart values for user, group and memory limit

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
Hey @gianmarco-mameli,

Thanks for the PR. Could you revert the runAsUser/Group values and we are good to go to merge.

File: charts/node-red/Chart.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ maintainers:
 annotations:
   artifacthub.io/containsSecurityUpdates: "false"
   artifacthub.io/changes: |
-    - update quay.io/kiwigrid/k8s-sidecar to 1.27.1
+    - Incorrect chart values for user, group and memory limit
```
Comment (dirien) [inline charts/node-red/Chart.yaml:32]:
```suggestion
    - Incorrect chart values for memory limit
```

**Review by dirien — CHANGES_REQUESTED** [review]

File: charts/node-red/values.yaml [hunk 2]
```diff
@@ -87,8 +87,8 @@ securityContext:
   runAsNonRoot: true
   allowPrivilegeEscalation: false
   readOnlyRootFilesystem: true
-  runAsGroup: 10003
-  runAsUser: 10003
+  runAsGroup: 1000
```
[the remark this answers is not in the document]
Comment (gianmarco-mameli) [reply charts/node-red/values.yaml:90]: Hi @dirien if you leave 10003 as user nodered raise an error when trying to do some operations, like clone a project. 
In my case the pod only works correctly with 1000 for uids.
If you want I can replicate the error and send you the pod log, in the meantime we can leave on hold the PR

Thanks

[the remark this answers is not in the document]
Comment (gianmarco-mameli) [reply charts/node-red/values.yaml:90]: As example: I'm unable to use git clone function to download the project that I need to run on node red
```
Cloning into 'k3stest'...
No user exists for uid 10003
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Maybe I can investigate more and send the PR like you suggested, let me know

[the remark this answers is not in the document]
Comment (gianmarco-mameli) [reply charts/node-red/values.yaml:90]: Hi @dirien, I've performed some tests and I get the same error, even trying to use 10000 as group and user.
The git clone inside the pod returns me
```
Cloning into '/data/projects/k3stest'...
    No user exists for uid 10000
```
Also similar problems are present in certain operation in nodered
No problem if I set 1000 as user.
Better to merge the pr without the mods on uid and gid, and maybe when I have a little more time I can check if there's a solution to use the chart with ids over 10000, in that case It's a problem confined in my environment

Thanks

**Comment by dirien** [comment]
hi @gianmarco-mameli,

can you please run `git commit --amend --signoff` to approve the DCO please?

**Comment by gianmarco-mameli** [comment]
Hi @dirien, no problem, I'm trying to solve the action on DCO but seems that your command, and the commands suggested from the DCO aren't working.
Tell me if I'm doing something wrong, thanks

**Comment by dirien** [comment]
@gianmarco-mameli 

I would say now rebase the main branch into this branch, then `git commit --amend --signoff` and then `git push origin HEAD --force` it. This should fix this.

**Comment by gianmarco-mameli** [comment]
I think it's still blocked, sorry for this mess

**Comment by dirien** [comment]
@gianmarco-mameli Fixed it.

**Pull request merged**